docs/index.md

```diff
@@ -33,8 +33,13 @@ Its rules look like the code you already write; no abstract syntax trees, regex
 The Semgrep ecosystem includes:
 
 * [Semgrep](getting-started/) - The open-source command line tool at the heart of everything.
+* [Semgrep Supply Chain](https://semgrep.dev/products/semgrep-supply-chain) - high-signal dependency scanner that detects reachable open source vulnerabilities in third-party libraries and functions across the SDLC.
```

Comment:
```suggestion
* [Semgrep Supply Chain](https://semgrep.dev/products/semgrep-supply-chain) - high-signal dependency scanner that detects reachable vulnerabilities in open source, third-party libraries and functions across the SDLC.
```

Just a nit/tiny suggestion, feel free to ignore. I get that we want to encompass all familiar terms with our users. But "open source vulnerabilities" sounds a little...strange? Where possible, btw, I've tried to stick to "dependencies" over "libraries" but I think it's fine to keep libraries here because dependency has already been mentioned.